src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionIntroViewController.swift

```diff
@@ -17,7 +17,7 @@
 
 import UIKit
 
-class ExposureSubmissionIntroViewController: DynamicTableViewController, ExposureSubmissionNavigationControllerChild, SpinnerInjectable {
+class ExposureSubmissionIntroViewController: DynamicTableViewController, ENANavigationControllerWithFooterChild, SpinnerInjectable {
```

Comment:
Is `SpinnerInjectable` necessary here?

Reply:
You tell me ;-)

Reply:
checked with @johannesrohwer , you can remove 
- `SpinnerInjectable` conformance
- `var spinner: UIActivityIndicatorView?`